tests/integration/tsconfig.issue952.json

```diff
@@ -1,8 +1,9 @@
 {
-  "extends": "./tsconfig.json",
+  "extends": "../../tsconfig.json",
   "compilerOptions": {
     "declaration": true,
     "emitDeclarationOnly": true
   },
-  "include": ["issue952.ts"]
+  "include": ["issue952.ts"],
+  "exclude": ["quick*.ts"]
```

Comment:
this one I'm going to address later by making a dedicated environment for this test